python/ray/serve/config.py

```diff
@@ -92,13 +93,18 @@ def replicas_settings_valid(cls, max_replicas, values):
 
         return max_replicas
 
-    @validator("policy", always=True)
```

Comment:
Validator is unable to run for attributes starting with underscore. So overriding init to call `serialize_policy` after init